persist/src/main/resources/db/migration/V14__AddContactProfilePicture.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE contacts ADD COLUMN profilePicture VARCHAR(500) DEFAULT null;
```

Comment:
formatting - each part of the statement should be on a new line 
```suggestion
ALTER TABLE contacts 
ADD COLUMN profilePicture VARCHAR(500) DEFAULT NULL;
```
also capitalize NULL in SQL